app/controllers/orders_controller.rb

```diff
@@ -8,7 +8,11 @@ class OrdersController < ApplicationController
   before_action :authorize_order
 
   def index
-    @orders = Order.all_or_today(params[:opt]).order(created_at: :desc)
+    if params[:user_id].present?
+      @orders = User.find(params[:user_id])&.orders&.all_or_today(params[:opt]).order(created_at: :desc)
```

Comment:
Missing last `&.`. Better to return redirect with a flash message if the user is not found.

Reply:
Fixed in 'error_handling' concern